core/src/main/java/com/microsoft/applicationinsights/internal/channel/samplingV2/FixedRateSamplingTelemetryProcessor.java

```diff
@@ -75,6 +76,7 @@ public FixedRateSamplingTelemetryProcessor() {
             }};
         } catch (ClassNotFoundException e) {
             InternalLogger.INSTANCE.trace("Unable to locate telemetry classes");
+            InternalLogger.INSTANCE.trace("stack trace is %s", ExceptionUtils.getStackTrace(e));
```

Comment:
Same as above (don't call the logger on the same level twice; combine the messages into one call)